libgphoto2_port/Makefile.am

```diff
@@ -96,6 +96,7 @@ noinst_DATA = libgphoto2_port-uninstalled.pc
 
 # ship headers
 nobase_include_HEADERS =	\
+	gphoto2/gphoto2-port-mutex.h			\
```

Comment:
Why install an internal header file (`gpi_*`)?

Reply:
hmm, we export it for libgphoto2 usage. but probably not needed there